content/metadata.yaml

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Manubot Rootstock: Manuscript Title'
+author_info:
+    -
+        github_username: dhimmel
+        full_name: Daniel S. Himmelstein
+        initials: DSH
+        orcid: 0000-0002-3012-7446
+        email: daniel.himmelstein@gmail.com
+        affiliations: Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania
+        funders: GBMF4552
+    -
+        github_username: agitter
+        full_name: Anthony Gitter
+        initials: AG
+        orcid: 0000-0002-5324-9833
+        email: gitter@biostat.wisc.edu
+        affiliations: Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison and Morgridge Institute for Research
+        funders: NIH U54AI117924
+keywords: ['work-in-progress', 'markdown', 'manuscript', 'publishing']
```

Comment:
I believe that it will also still work if leave out the quotation marks, which is nice.